google/cloud/sql/integration_tests/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+# ~~~
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ~~~
+
```

Comment:
Since we wrote these files I learnt we can simplify them with:

```
if (NOT BUILD_TESTING)
  return()
endif ()

# skip the function, just put the code as normal.

```

Reply:
We can even skip the if statement, because we only `add_subdirectory` if `BUILD_TESTING`